docs/shadow.md

```diff
@@ -30,6 +30,31 @@ This uses custom elements, `my-app`, `recipe-hello`, `recipe-hello-binding` and
 
 ## Locate Shadow Dom
 
+### Option 1: Use the [query-selector-shadow-dom](https://github.com/Georgegriff/query-selector-shadow-dom) plugin
```

Comment:
@Georgegriff could you verify this part as your plugin is mentioned )

Reply:
Yep it's on my to-do list